draft-ietf-taps-interface.md

```diff
@@ -1438,6 +1436,11 @@ more than once for a single Send Action. It is used to mark data safe for
 certain 0-RTT establishment techniques, where retransmission of the 0-RTT data
 may cause the remote application to receive the Message multiple times.
 
+Note that for protocols that do not protect against duplicated messages,
+e.g., UDP, all messages MUST be marked as Idempotent. 
+In order to enable protocol selection to choose such a protocol, 
+Idempotent MUST be added to the TransportProperties passed to the Preconnection.
```

Comment:
This is good, but I think it's important to add something like this: "If Idempotent has been chosen as a default for the Preconnection, it may not be possible to enable it per Message later."

I mean, once UDP has been chosen, disabling Idempotent can't work, right?  So if the transport system still offers it, this produces a wrong expectation for an application. We should think if there are other Message Properties that have the same kind of behavior. I guess so - what if reliability = off by default? We MAY be able to enable it later (e.g. if SCTP with partial reliability was chosen, or even TCP, simply sending everything reliable), but it can't be promised (e.g. if UDP was chosen).

Reply:
If you disable idempotent later, this may or may not work (as most message properties) I will add a hint here.